src/main/java/com/commercetools/sync/inventory/utils/InventorySyncUtils.java

```diff
@@ -0,0 +1,77 @@
+     * @param syncOptions the sync options wrapper which contains options related to the sync process supplied by the user.
+     *                    For example, custom callbacks to call in case of warnings or errors occurring on the build
+     *                    update action process. And other options (See {@link BaseSyncOptions}
+     *                    for more info.
+     * @param typeService responsible for fetching the key of the old resource type from it's cache.
+     * @return list containing {@link UpdateAction} that need to be performed on {@code oldEntry} resource so
+     * that it will be synced with {@code newEntry} or empty list when both entries are already synced.
+     */
+    @Nonnull
+    public static List<UpdateAction<InventoryEntry>> buildActions(@Nonnull final InventoryEntry oldEntry,
+                                                                  @Nonnull final InventoryEntry newEntry,
+                                                                  @Nonnull final InventorySyncOptions syncOptions,
+                                                                  @Nonnull final TypeService typeService) {
+        final InventoryEntryDraft newEntryDraft = transformToDraft(newEntry);
+        return buildActions(oldEntry, newEntryDraft, syncOptions, typeService);
+    }
+
+    /**
+     * This method returns list of {@link UpdateAction} that needs to be performed on {@code oldEntry} resource so
+     * that it will be synced with {@code newEntry} draft.
+     *
+     * @param oldEntry entry which data should be updated
+     * @param newEntry draft that contain current data (that should be applied to {@code oldEntry}
+     * @param syncOptions the sync options wrapper which contains options related to the sync process supplied by the user.
+     *                    For example, custom callbacks to call in case of warnings or errors occurring on the build
+     *                    update action process. And other options (See {@link BaseSyncOptions}
+     *                    for more info.
+     * @param typeService responsible for fetching the key of the old resource type from it's cache.
+     * @return list containing {@link UpdateAction} that need to be performed on {@code oldEntry} resource so
+     * that it will be synced with {@code newEntry} or empty list when both entries are already synced.
+     */
+    @Nonnull
+    public static List<UpdateAction<InventoryEntry>> buildActions(@Nonnull final InventoryEntry oldEntry,
+                                                                  @Nonnull final InventoryEntryDraft newEntry,
+                                                                  @Nonnull final InventorySyncOptions syncOptions,
+                                                                  @Nonnull final TypeService typeService) {
+        final List<UpdateAction<InventoryEntry>> actions = new LinkedList<>();
```

Comment:
I was not aware of that internal conversion. It's fixed now 5d41c51c1f568a1f7f918b742ff01d04d916a86e